Index: app/src/main/java/ua/dokser/calculatorforinotropesfornewborn/MainActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ua/dokser/calculatorforinotropesfornewborn/MainActivity.kt b/app/src/main/java/ua/dokser/calculatorforinotropesfornewborn/MainActivity.kt
--- a/app/src/main/java/ua/dokser/calculatorforinotropesfornewborn/MainActivity.kt	
+++ b/app/src/main/java/ua/dokser/calculatorforinotropesfornewborn/MainActivity.kt	(date 1682009745966)
@@ -3,35 +3,29 @@
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.util.Log
+import android.widget.TextView
 import ua.dokser.calculatorforinotropesfornewborn.databinding.ActivityMainBinding
 import kotlin.math.roundToInt
 
 class MainActivity : AppCompatActivity() {
+
+
+//    var weight:Double? = 0.0
+//    var dose_of_inotrop: Double? = 0.0
+//    var infusion_rate:Double? = 0.0
+//    var concentration_of_inotrop: Double? = 0.0
+//    var volume_of_syringe = 0
+//    var  result:Double = 0.0
+
     lateinit var binding: ActivityMainBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
-        binding.apply {
-            val weight = 
-
-        }
-        binding.resultBtn.setOnClickListener {
-            if (){
-                calculateOfInotropes()
-            }
-        }
     }
 }
-private fun calculateOfInotropes(
-    weight:Double,
-    dose_of_inotrop: Double,
-    infusion_rate: Double,
-    concentration_of_inotrop: Double,
-    volume_of_syringe: Int) : Double {
-    var result = (weight*dose_of_inotrop*6*volume_of_syringe)/(infusion_rate*concentration_of_inotrop*1000)
-    Log.d("MyLog","Result= $result")
-    result = (result*100).roundToInt()/100.0
-    return result
-    }
+
+
+
